emailing-service/emailing-service/Models/EmailType/RawEmailModel.cs

```diff
@@ -0,0 +1,27 @@
+namespace emailing_service.Models.EmailType;
+
+public class RawEmailModel
+{
+    public RawEmailModel()
```

Comment:
If you permit it, can I ask why there is an empty constructor ?

Reply:
Of course! Let me enlighten you. The empty constructor only was used in the context of a test in order to check how the application would resolve if it was given an empty RawEmailModel!

Reply:
That explains it!